pom.xml

```
@@ -16,17 +16,14 @@
   <description>Kogito Apps</description>
 
   <properties>
-    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
     <version.frontend.plugin>1.9.1</version.frontend.plugin>
     <version.node>v12.16.2</version.node>
     <version.npm>6.10.3</version.npm>
     <version.yarn>v1.19.1</version.yarn>
     <maven.compiler.release>11</maven.compiler.release>
     <version.maven-remote-resources-plugin>1.7.0</version.maven-remote-resources-plugin>
 
-    <version.org.infinispan>10.1.5.Final</version.org.infinispan>
     <version.org.infinispan.image>10.1.5.Final</version.org.infinispan.image>
```

Let's update the images too
```suggestion
    <version.org.infinispan.image>${version.org.infinispan}</version.org.infinispan.image>
```
or 
```suggestion
    <version.org.infinispan.image>11.0.1.Final</version.org.infinispan.image>
```

---

@danielezonca that will break :) There is a story just to handle all that is needed to do that https://issues.redhat.com/browse/KOGITO-3118